documents/read_acls.go

```diff
@@ -388,11 +388,7 @@ func (cd *CoreDocument) ATGranteeCanRead(ctx context.Context, docService Service
 		return ErrReqDocNotMatch
 	}
 	// validate that the public key of the granter is the public key that has been used to sign the access token
-	doc, err := docService.GetVersion(ctx, cd.Document.DocumentIdentifier, at.DocumentVersion)
```

Comment:
lets keep it the way before as this is more secure as the logic is encapsulated to check and get the model version by it self for the given AT